src/components/organisms/Video/index.ts

```diff
@@ -1 +1 @@
-export { VideoParticipant } from "./VideoParticipant";
+export const hello = 1;
```

Comment:
hey
do we still need it?

Reply:
why is hello 1? can't you have 2 hellos? 🤪

Reply:
Resolved in 03a5d4919